Swiftfin tvOS/Views/LiveTVChannelsView.swift

```diff
@@ -55,15 +55,21 @@ struct LiveTVChannelsView: View {
 		let channel = item.channel
 		if channel.type != "Folder" {
 			Button {
+                self.viewModel.isLoading = true
 				self.viewModel.fetchVideoPlayerViewModel(item: channel) { playerViewModel in
 					self.router.route(to: \.videoPlayer, playerViewModel)
+                    DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
```

Comment:
Sorry about this. I don't like it, but without it, the loading view dismisses a fraction too early, so the LiveTV channels grid flashes visible before the player view loads. That experience felt janky. This code also feels janky... but it fixes it from the viewer's perspective.